dev/io.openliberty.ws.jaxws.global.handler.internal_fat/fat/src/com/ibm/ws/webservices/handler/fat/TestUtils.java

```diff
@@ -227,15 +228,19 @@ public static String getBaseTestUri(String contextRoot) {
     public static void installUserFeature(LibertyServer server, String featureManifestFile) throws Exception {
         if (JakartaEE9Action.isActive()) {
             server.installUserFeature("ee9/" + featureManifestFile);
+        } else if (JakartaEE10Action.isActive()) {
+            server.installUserFeature("ee10/" + featureManifestFile);
         } else {
             server.installUserFeature(featureManifestFile);
         }
     }
 
     public static void setServerConfigurationFile(LibertyServer server, String newConfigFile) throws Exception {
-        if (JakartaEE9Action.isActive()) {
+        if (JakartaEE9Action.isActive() || JakartaEE10Action.isActive()) {
```

Comment:
This seems wrong.  The .ee10 versions of the files are missing in the PR.

Reply:
Yes, that is my mistake, I'm fixing.